src/wrapped.rs

```diff
@@ -121,11 +121,17 @@ fn package_tar_review(name: &str, dirs: &ProjectDirs) {
 	});
 }
 
-// TODO: use libalpm.uname instead of machine uname
```

Comment:
Yes. I saw it before I started to code. But the last commit to this repo was from 20 Aug 2017. So I decided to not pull this in as additional dependency for `rua`.

Reply:
@senden9 I see your concerns.
Currently, RUA uses it now to determine whether packages are installed, whether they are installable (e.g. a package group, which is not technically a package) etc.

There's a fork which ported libalpm to stable Rust (mid-2018). For us this is good news as it can only break if libalpm itself makes incompatible changes. By the way, now that you reminded me, it may be a good time to ping the original author about the fork -- he initially wanted to merge it back.